managed/services/agents/vmagent.go

```diff
@@ -16,19 +16,31 @@
 package agents
 
 import (
+	"os"
 	"sort"
 
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
 )
 
+var (
+	maxScrapeSizeEnv     = "PMM_PROMSCRAPE_MAX_SCRAPE_SIZE"
+	maxScrapeSizeDefault = "64MiB"
+)
+
 // vmAgentConfig returns desired configuration of vmagent process.
 func vmAgentConfig(scrapeCfg string) *agentpb.SetStateRequest_AgentProcess {
+	maxScrapeSize := maxScrapeSizeDefault
+	if space := os.Getenv(maxScrapeSizeEnv); space != "" {
+		maxScrapeSize = space
+	}
```

Comment:
How server behave on invalid string here? Is it clear from logs what is the problem?

Reply:
INFO[2023-03-13T11:46:08.732+01:00] invalid value "64ABC" for flag -promscrape.maxScrapeSize: strconv.ParseFloat: parsing "64ABC": invalid syntax  agentID=/agent_id/c717a23c-bfdd-4c36-9723-5d4a6388fde5 component=agent-process type=vm_agent